### ISSUE TITLE
Possibility of a smaller certificate?

### ISSUE THREAD

**Issue by dominikponniah** [admin-ch/CovidCertificate-Documents#25]
Especially older people who have devices with KaiOS or other operating systems without Android are not able to use the Covid Certificate app.

Carrying the A4 page around is also not up-to-date and suboptimal. Is it legally permissible to cut out the QR codes and stick them on the back of a Cumulus card (or bank card), for example?

**Comment by delixfe** [comment]
Yes, you can cut out the QR code. Please use a copy for this, so that you do not have to request a certificate if the QR code is not readable any more. 

You could also present a good photo of the QR code, as all the data necessary for the verification is contained in the QR code. 

Please keep in mind, that you also need to show an official ID card or other suitable official document.

**Comment by dominikponniah** [comment]
Thanks @delixfe,
I developed a simple Webapp which should create a 'CovidCard' using the Official QR-Code to print this out in CreditCard-Format. I assume that this would also work, right? Or is this a 'Zweckentfremdung des Zertifikates'?

Cheers!

**Comment by gagarine** [comment]
I'm not familiar with KaiOS devices, but if you can display stored picture you can also keep a photographie of the QR code. Try to scan it to be sure it works with the KaiOS resolution screen.

**Comment by ypiguet-epfl** [comment]
@delixfe
Technically a QR-code with valid certificate data (light or not) and the electronic signature should be fine. But the communication from BAG is more restrictive: section 1 of [FAQ – Prüfung der Covid-Zertifikate](https://www.newsd.admin.ch/newsd/message/attachments/68147.pdf) (or [French](https://www.newsd.admin.ch/newsd/message/attachments/68148.pdf) or [English](https://www.newsd.admin.ch/newsd/message/attachments/68154.pdf)) require to scan the code _on the paper certificate or in the COVID Certificate app_. We cannot expect everybody to fully understand how QR-codes work and take liberties with the official instructions. Presenting anything else (just the QR-code on a bit of paper or full-screen on a dumbphone) would be risky.

**Comment by delixfe** [comment]
@ypiguet-epfl
Thanks for your input. I have forwarded the issue.

**Comment by delixfe** [comment]
Sorry, for the late feedback. The documentation was adapted. See https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/covid-zertifikat/covid-zertifikat-pruefer-aussteller-technische-informationen.html.
The PDFs referenced before were not changed, but they are not linked anymore on the website.
